.github/workflows/backport.yaml

```diff
@@ -15,26 +15,19 @@ jobs:
     if: >
       github.event.pull_request.merged
       && (
-          (github.event.action == 'closed' && contains(github.event.pull_request.labels.*.name, 'backport/'))
```

Comment:
I reverted this part of the trigger because it resulted in not triggering when the PR was closed.